skrifa/src/instance.rs

```diff
@@ -50,6 +52,15 @@ impl Size {
             1.0
         }
     }
+
+    /// Computes a fixed point linear scale factor that matches FreeType.
+    pub(crate) fn fixed_linear_scale(self, units_per_em: u16) -> Fixed {
```

Comment:
Would be nice to link to the corresponding FT code and explain the rationale if it's not obvious from the corresponding FT code. Currently it's a wee bit magical.

Reply:
Wee bit is probably an understatement :) Will do.